src/worker.py

```
@@ -40,11 +40,201 @@
 import re
 import traceback
 from types import SimpleNamespace
+import uuid
+from workers import DurableObject, Response
+class WaitingRoomDO(DurableObject):
+    def __init__(self, state, env):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)